RELEASE/scripts/autoscend.ash

```diff
@@ -1557,16 +1557,7 @@ boolean autosellCrap()
 	{
 		return false;		//selling things in the way of the surprising fist only donates the money to charity, so we should not autosell anything automatically
 	}
-	foreach it in $items[dense meat stack, meat stack,  //quest rewards that are better off as meat. If we ever need it we can freely recreate them at no loss.
-	Blue Money Bag, Red Money Bag, White Money Bag,  //vampyre path boss rewards and major source of meat in run.
-	Space Blanket, //can be inside MayDay package. Only purpose is to sell for meat
-	Void Stone] //dropped by Void Fights when Cursed Magnifying Glass is equiped. Only purpose is to sell for meat
-	{
-		if(item_amount(it) > 0)
-		{
-			auto_autosell(min(10,item_amount(it)), it);		//autosell all of this item
-		}
-	}
+
```

Comment:
```
meat stack,
    Blue Money Bag, Red Money Bag, White Money Bag,  //vampyre path boss rewards and major source of meat in run.
	Space Blanket, //can be inside MayDay package. Only purpose is to sell for meat
	Void Stone] //dropped by Void Fights when Cursed Magnifying Glass is equiped. Only purpose is to sell for meat
```
have disappeared entirely and do not appear to be sold

Reply:
good catch, added back in.